apidef/oas/example_test.go

```diff
@@ -93,7 +97,7 @@ func Test_exampleExtractor(t *testing.T) {
 			`{
 				"type": "string",
 				"example": "bird",
-				"enum": ["duck"]	
+				"enum": ["duck", "bird"]	
```

Comment:
I didn't get this why it changes here?

Reply:
example contains `bird` hence adding it to enum